modules/standard/Regexp.chpl

```diff
@@ -351,11 +735,16 @@ record regexp {
   }
 
 
-  /* Split the text by occurrences of this regular expression.
+  /*
+     Split the text by occurrences of this regular expression.
      If capturing parentheses are used in pattern, then the text of all
      groups in the pattern are also returned as part of the resulting array.
      If maxsplit is nonzero, at most maxsplit splits occur, and the
```

Comment:
Might want to italicize the references to maxsplit here, so there's more of a visual cue that you're referring to an argument.  No requirement to do so, though.